travis/setup_xspec_ds9.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Tarballs to fetch
+DS9_SITE=http://ds9.si.edu/download/centos6/;
+XPA_SITE=http://ds9.si.edu/download/centos6/;
+DS9_TAR=ds9.centos6.7.5.tar.gz;
+XPA_TAR=xpa.centos6.2.1.17.tar.gz;
+
+# Fetch them
+wget --quiet $DS9_SITE$DS9_TAR;
+wget --quiet $XPA_SITE$XPA_TAR;
+
+# untar them
+THIS_DIR=$(pwd);
+cd $MINICONDA/bin; tar xf $THIS_DIR/$DS9_TAR; tar xf $THIS_DIR/$XPA_TAR; cd -;
+
+# install build dependencies
+sudo apt-get update
+sudo apt-get install -qq libwcs4 wcslib-dev libx11-dev libsm-dev libxrender-dev;
+
+# Some relevant environment variables
+export HEADAS=$MINICONDA/envs/build/Xspec/spectral;
```

Comment:
Shouldn't HEADAS be set to .../envs/build/Xspec/spectral/x85_64-blah-blah-libc2.15.0?

See step 8 of https://heasarc.gsfc.nasa.gov/lheasoft/install.html

In which case XSPEC_INCLUDE_PATH is just ${HEADAS}/include/

Reply:
Maybe I am missing something... `$MINICONDA/envs/build/Xspec/spectral` only contains the `manager` and `modelData` folders, so indeed it's what `HEADAS` should point to. While the include files are not there, they are located in the platform-dependent folder. Am I missing something?

Reply:
I think it's different with models-only, `HEADAS` is the environment variable that the Sherpa extension needs to point to the folder containing `manager` and `modelData`. That's how we've always done it in Sherpa. We don't run the init script and we don't require users to run it. There used to be instructions on how to build the models-only package, the document seems to be gone now (https://heasarc.gsfc.nasa.gov/docs/software/lheasoft/xspec-models-only.txt), I wouldn't change the way we've done things so far, but maybe we should double-check our installation instructions work?